packages/aragon-wrapper/src/identity/AddressBookIdentityProvider.js

```diff
@@ -0,0 +1,64 @@
+import AddressIdentityProvider from './AddressIdentityProvider'
+import { first, map } from 'rxjs/operators'
+import { getCacheKey } from '../utils/index'
+
+const addressBookAppIds = [
```

Comment:
we have to actually put this in the javascript? does that mean each time we republish the contract, we need to submit a PR?

Reply:
As long as we publish to the same namespaces, this will not need to change